chapters/en/slides/module2/module2_29.Rmd

```diff
@@ -153,21 +171,24 @@ Notes: Script here
 
 ---
 
-In the last plot, we used `.loc[:,'sugars']` to select a single column to the plot, however, we can show multiple mean column values in a single plot by selecting more columns. The columns `fat`, `fiber` and `protein` seem like good choices. 
-
-```{python out.width = '60%', fig.asp = .58}
-nutrition_plot = (df.groupby(by='mfr')
-                    .mean()
-                    .loc[:, ['fat', 'fiber', 'protein']]
-                    .plot.bar(title='Mean nutritrion value over different manufacturers')
-                 )
-nutrition_plot.set_ylabel('Content (in grams)', fontsize=9)
-nutrition_plot = nutrition_plot.set_xlabel('Manufacturer', fontsize=9)
-nutrition_plot
+The `x` and `y` labels from the bar plot on the previous slide are not very informative. Luckily, using the same syntax, we can set custom axis `titles` using the `title=""` argument. Let's see how this works by providing more informative titles to the plot from the previous slide.
+
+```{python}
+chart4 = alt.Chart(cereal, width=500, height=300).mark_bar().encode(
+    x=alt.X('calories:Q', bin=alt.Bin(maxbins=50), title="Calory content"), # Change the axis title
+    y=alt.Y('count():Q', title="Count") # Change the axis title
+).properties(title="Bar plot of cereal calory content")
+chart4
+
 ```
 
 
-If you want high fibre and low fat, consider having N's cereals for breakfast (or lunch or dinner)!
```

Comment:
This to keep this in if possible it was recommended by Mike ( in the plot associated with it)